### ISSUE TITLE
React.createElement: type should not be null, undefined, boolean, or number

### ISSUE THREAD

**Issue by hindsc52** [nfl/react-helmet#173]
Hi, When importing and including this component on my page I receive the following errors:

`React.createElement: type should not be null, undefined, boolean, or number`

`bundle.js:1 Error: Element type is invalid: expected a string (for built-in components) or a class/function`

These error only appear when i an using the react-helmet component.

Here is my full component so far:

```
import Helmet from 'react-helmet';
import React from 'react';
import grandstand from 'grandstand';

export default class MyModule extends React.Component {
    constructor(props) {
        super(props);

        // initial state
    }

    componentWillMount() {
        grandstand(this.props.serviceName);
        console.log(Helmet);
    }

    render() {
        return (
            <div>
                <Helmet title="My Title"/>
                <h1>Welcome to the mobile web</h1>
            </div>
        )
    }
}

MyModule.displayName = 'MyModule';

MyModule.propTypes = {
    serviceName: React.PropTypes.string
};
```

**Comment by dlebedynskyi** [comment]
@hindsc52  started to have simmilar issues today. 
```
Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object. Check the render method of `App`.
```
with 
![image](https://cloud.githubusercontent.com/assets/1182917/20064802/f2c10c10-a4d9-11e6-8388-c86848c5ffdc.png)

**Comment by sjzamora86** [comment]
I'm having same issue when doing unit test using Enzyme full rendering

**Comment by cwelch5** [comment]
@hindsc52 This could be because you are console logging Helmet _before_ the Helmet is mounted.  If you remove the console do you still get the error?

**Comment by cwelch5** [comment]
@hindsc52 @sjzamora86  I can't seem to reproduce with the console log.  And can't reproduce in my unit tests, using Karma though.

Do either of you see this issue in another component with Helmet?  Or any more detail you can add to your experience?

**Comment by hindsc52** [comment]
@cwelch5 Hi unfortunately i had stopped working on the project using this component so can no longer comment on the issue.

**Comment by potench** [comment]
Closing this issue since this thread is cold. Issue may be related to https://github.com/nfl/react-helmet/issues/190, so if we need to re-open this issue please post what version of Helmet you're using as this is fixed in v3.2.2
